modules/ROOT/pages/questions/question-04.adoc

```diff
@@ -0,0 +1,13 @@
+4. The `toc` plugin has been documented as _new and improved_ because
+   of the following:
+
+   *the `toc` plugin has modernised its output. It now uses the ES2020
```

Comment:
@lnewson

Reply:
This is a generic change done across all plugins and core. It's just that we no longer support IE 11 so we've modernized the output/requirements to run TinyMCE. It now needs a browser that's capable of running ES2020 which is actually all our supported browsers.

So basically this isn't something we need to mention in the release notes for Table of Contents specifically, it's a much larger change that impacts all of TinyMCE 6.0.